### PR TITLE
Changed nonce check from signatory to signer

### PR DESCRIPTION
**Pull request by smartcontrart**

### CODE REVIEW

File: source/swap/contracts/Swap.sol [hunk 1]
```diff
@@ -495,21 +492,21 @@ contract Swap is ISwap, Ownable, EIP712 {
...
    */
-  function _markNonceAsUsed(address signer, uint256 nonce) internal {
+  function _markNonceAsUsed(address signatory, uint256 nonce) internal {
     uint256 groupKey = nonce / 256;
     uint256 indexInGroup = nonce % 256;
-    uint256 group = _nonceGroups[signer][groupKey];
+    uint256 group = _nonceGroups[signatory][groupKey];
 
-    // If it is already used, return mit cancel and revert
+    // If it is already used, return cancel and revert
```
Comment (dmosites) [inline source/swap/contracts/Swap.sol:504]: // Revert if nonce is already used

**Review by dmosites — CHANGES_REQUESTED** [review]
Looks good, small comment change requested.

File: source/swap/contracts/Swap.sol (hunk 1 above)
Comment (smartcontrart) [reply source/swap/contracts/Swap.sol:504]: agreed

**Review by dmosites — APPROVED** [review]
LGTM

**Pull request merged**